aces/Loris
# [Integration Test] Update Genomic Browser filter tests

Updated the Genomic Browser integration tesst to use the new _filterTest method

This PR still needs tests for filters on other tabs of the Genomic Browser module

Testing instructions
Check out Travis for integration test results or use the following command to run the test on terminal:

`npm run tests:integration -- --filter GenomicBrowserTestIntegrationTest::testGenomicBrowserFiltersProfileTab`
